cpp/src/com/google/idea/blaze/cpp/BlazeCWorkspace.java

```diff
@@ -257,6 +355,9 @@ private OCWorkspaceImpl.ModifiableModel calculateConfigurations(
                 .map(file -> "-I" + file.getAbsolutePath())
                 .collect(toImmutableList());
 
+        Path rootPath = workspaceRoot.directory().toPath();
+        List<String> includes = collectIncludesWithProgress(rootPath, targetKey, blazeProjectData, indicator);
```

Comment:
these `includes` are quite specific ones. should we say something like `collectBazelStripIncludeHints`?